recipe/meta.yaml

```diff
@@ -11,25 +11,41 @@ source:
   url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
 
 build:
-  skip: true  # [win]
   number: 3
   binary_relocation: true
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
 
 requirements:
   build:
     - cmake
     - toolchain
+    - python  # [win]
 
 test:
+  requires:
+    - python 2.7.*  # [win and py27]
+    - python 3.4.*  # [win and py34]
+    - python 3.5.*  # [win and py35]
```

Comment:
You need to do what I mention in https://github.com/conda-forge/metis-feedstock/pull/7#discussion_r87870092 for that to work properly. Sorry if I was not clear before.